util/separators.lua

```diff
@@ -18,14 +18,24 @@ local separators = { height = 0, width = 9 }
 -- Right
 function separators.arrow_right(col1, col2)
     local widget = wibox.widget.base.make_widget()
+    widget.col1 = col1
+    widget.col2 = col2
 
     widget.fit = function(m, w, h)
         return separators.width, separators.height
     end
 
+    widget.update = function(col1, col2)
+        widget.col1 = col1
+        widget.col2 = col2
+        local vis = widget:get_visible()
+        widget:set_visible(not vis)
+        widget:set_visible(vis)
```

Comment:
this goofiness is purely just to get the widget to redraw, if there's a better way I'd love to know

Reply:
It should be `widget:emit_signal("widget::redraw_needed")`.